content/master/concepts/providers.md

```diff
@@ -3,52 +3,306 @@ title: Providers
 weight: 101
 ---
 
-Providers are Crossplane packages that bundle a set of 
-[Managed Resources]({{<ref "managed-resources" >}}) and their respective controllers to allow
-Crossplane to provision the respective infrastructure resource.
+Providers enable Crossplane to provision infrastructure on an
+external service. Providers create new Kubernetes APIs and map them to external 
+APIs.
 
-## Installing Providers
+Providers are responsible for all aspects of connecting to non-Kubernetes
+resources. This includes authentication, making external API calls and
+providing
+[Kubernetes Controller](https://kubernetes.io/docs/concepts/architecture/controller/)
+logic for any external resources.
 
-The core Crossplane controller can install provider controllers and CRDs for you
-through its own provider packaging mechanism, which is triggered by the
-application of a `Provider` resource. For example, in order to request
-installation of the `provider-aws` package, apply the following resource to the
-cluster where Crossplane is running:
+Examples of providers include:
+* [Provider AWS](https://github.com/upbound/provider-aws)
+* [Provider Azure](https://github.com/upbound/provider-azure)
+* [Provider GCP](https://github.com/upbound/provider-gcp)
+* [Provider Kubernetes](https://github.com/crossplane-contrib/provider-kubernetes)
 
-```yaml
+{{< hint "tip" >}}
+Find more providers in the [Upbound Marketplace](https://marketplace.upbound.io).
+{{< /hint >}}
+
+<!-- vale write-good.Passive = NO -->
+<!-- "are Managed" isn't passive in this context -->
+Providers define every external resources they can create in Kubernetes as a
+Kubernetes API endpoint. These endpoints are 
+[_Managed Resources_]({{<ref "managed-resources" >}}).
+<!-- vale write-good.Passive = YES -->
+
+Providers are cluster scoped and any namespace can access them.
```

Comment:
What are you intending to communicate here?

* If you mean the `Provider` API type that is used to install providers, it's more accurate to say they exist outside of any namespace. Namespaces don't really "access" them, and callers aren't accessing them "from" a namespace.
* If you mean the provider in the more general sense, i.e. the running process that reconciles stuff, they can't be accessed from any namespace.